parser.go

```diff
@@ -33,8 +34,18 @@ func parse(m *Message, channel *ChannelData, user *User) (*Cmd, error) {
 		return nil, nil
 	}
 
+	// split command from args by unicode space
+	firstOccurrence := true
+	f := func(c rune) bool {
```

Comment:
```suggestion
	firstOccurrence := true
	firstUnicodeSpace := func(c rune) bool {
```
This says the same as the comment.

Reply:
My suggestion was to rename `f` to `firstUnicodeSpace`, not just remove the comment, this way it is clear what the function does.

Reply:
Ah I see sorry I missed that, will update now

Reply:
Thanks!